src/components/Navigation/DesktopNavigation/index.js

```diff
@@ -0,0 +1,55 @@
+import { Section } from "@commons-ui/core";
+import { Grid } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import PropTypes from "prop-types";
+import React from "react";
+
+import Logo from "@/pesayetu/components/Logo";
+import Menu from "@/pesayetu/components/Menu";
+
+const useStyles = makeStyles(({ typography }) => ({
+  root: {
+    padding: `${typography.pxToRem(9.25)} 0`,
+  },
```

Comment:
We don't need the extra padding here. 

Total height on Desktop should be `100`px, we are at `125`px